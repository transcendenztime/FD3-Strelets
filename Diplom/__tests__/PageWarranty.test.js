import React from 'react';
import renderer from 'react-test-renderer';
import { BrowserRouter as Router } from 'react-router-dom';

import PageWarranty from '../pages/PageWarranty';

test('внешний вид PageWarranty', () => {

    // создаём тестовую версию компонента
    const component = renderer.create(
      <Router>
          <PageWarranty />
      </Router>
    );
  
    // получаем снэпшот (HTML-снимок) компонента для сверки, что вёрстка не испортилась
    let componentTree=component.toJSON();
    expect(componentTree).toMatchSnapshot();
          
  });