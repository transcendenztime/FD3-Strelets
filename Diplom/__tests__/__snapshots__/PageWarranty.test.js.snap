// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`внешний вид PageWarranty 1`] = `
<div
  className="warranty page"
>
  <h1
    className="page-title"
  >
    Гарантии:
  </h1>
  <div
    className="breadcrumbs-container"
  >
    <a
      aria-current={null}
      className="breadcrumbs"
      href="/"
      onClick={[Function]}
      style={undefined}
    >
      Главная 
    </a>
    <span>
       → 
    </span>
    <a
      aria-current={null}
      className="breadcrumbs"
      href="/warranty"
      onClick={[Function]}
      style={undefined}
    >
      Гарантии
    </a>
  </div>
  <hr />
  <section
    className="description"
  >
    <p>
      Мы предоставляем гарантии на весь посадочный материал, предлагаемый в каталоге нашего сайта.
    </p>
    <blockquote>
      <p>
        <b>
          Любой садовод знает, что растения - это живые существа, большое влияние на которые оказывает свет, влага, температура воздуха и множество других факторов.
        </b>
      </p>
    </blockquote>
    <p>
      Мы используем максимально надежные методы упаковки, но наши товары - это растения, и во время пути с посадочным материалом могут произойти необратимые изменения, особенно если не были соблюдены условия перевозки. Нас очень огорчают такие редкие случаи, но мы всегда гарантируем качество своим клиентам и делаем все, чтобы исправить ситуацию. 
    </p>
    <blockquote>
      <p>
        <b>
          Если за время доставки товар пришел к Вам в неподходящем виде, Вам нужно составить претензию и мы найдем решение проблемы в индивидуальном порядке.
        </b>
      </p>
    </blockquote>
    <p>
      <b>
        Все претензии принимаются в течении 14 дней с момента получения заказа в письменной форме
      </b>
      :
    </p>
    <ul>
      <li>
        на почтовый Адрес: Беларусь, индекс 220006, а/я 282
      </li>
      <li>
        или по электронной почте E-mail: 
        <a
          href="mailto:info@dolinasad.by"
        >
          info@dolinasad.by
        </a>
      </li>
    </ul>
    <p>
      В письме необходимо указать полное ФИО, описать в каком состоянии Вы получили посылку, наличие дефектов внешнего вида посылки, к письму приложить фотографии растения. Обязательно напишите, какую именно компенсацию вы хотите – замену на другой товар, денежную компенсацию или нужно просто выслать Вам аналогичные семена или саженцы, взамен испорченных.
    </p>
    <p>
      <b>
        Пересылка замены осуществляется за НАШ счет.
      </b>
    </p>
  </section>
</div>
`;
